libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFNetworkPlugin/SFNetworkPlugin.m

```diff
@@ -57,18 +58,27 @@ - (void) pgSendRequest:(CDVInvokedUrlCommand *) command
     NSString* path = [argsDict nonNullObjectForKey:kPathArg];
 
     NSDictionary* queryParams = [[NSDictionary alloc] init];
+    NSDictionary<NSString*, NSString*>* headerParams = [argsDict nonNullObjectForKey:kHeaderParams];
```

Comment:
We need the ability to specify any http request headers (custom end point could expect specific headers and the client should have a way to specify them or for instance if-xxx headers for "smarter" end points). This is not the artifact of old functionality.

Content type is one of many "permanent"  http request headers (full list here: https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Request_fields), I don't see a reason to single it out and treat it as a separate parameter.